.github/workflows/gh-pages-main.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will generate the static page under `main` subdirectory inside the `gh-pages` branch
```

Comment:
Suggestion: Wrap "non-essential" lines at approx. 80 characters. This is ... traditional, but flexible.
And opinions vary, but see: https://stackoverflow.com/a/578318

This is what I see when I try to review, and the horizontal scroll bar is not always visible
![image](https://user-images.githubusercontent.com/20852/171513444-74a193fe-dec7-499e-8f7b-ca0f4e5062ad.png)

Reply:
My VSCode has "ghost" vertical lines at 80 and 120 characters, I think by default.

Reply:
I also want to encourage, but not insist: Write shorter while staying clear: 

Original: 
```
# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
```
Possible: 
```
- Runs tests,  _this happens_ when fail
- Publishes _package_named_by?_ to GitHub Packages
-- Tagged by ...
-- Given release version by ... (incrementing ...  alpha ...)